src/CMakeLists.txt

```diff
@@ -542,8 +539,14 @@ set_target_properties(${target} PROPERTIES
 # packaging
 set(PACKAGE_NAME ${PROJECT_NAME})
 
+set(QSK_EXPORTED_TARGETS ${target})
```

Comment:
I know I suggested it, but lets stick to local variables `lower_snake_case` and preserve `QSK_UPPER_CASE` for "global" variables. So please rename to e.g. `exported_targets`